code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm

```diff
@@ -0,0 +1,34 @@
+/**
+  *	Lobstrosities, the poster boy of charging AI mobs. Drops crab meat and bones.
+  * Outside of charging, it's intended behavior is that it is generally slow moving, but makes up for that with a knockdown attack to score additional hits.
+  */
+/mob/living/simple_animal/hostile/asteroid/lobstrosity
+	name = "lobstrosity"
+	desc = "A marvel of evolution gone wrong, the frosty ice produces underground lakes where these ill tempered seafood gather. Beware its charge."
+	icon = 'icons/mob/icemoon/icemoon_monsters.dmi'
+	icon_state = "lobstrosity"
+	icon_living = "lobstrosity"
+	icon_dead = "lobstrosity_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mouse_opacity = MOUSE_OPACITY_ICON
+	friendly_verb_continuous = "chitters at"
+	friendly_verb_simple = "chits at"
+	speak_emote = list("clatters")
+	speed = 3
+	move_to_delay = 20
+	maxHealth = 100
```

Comment:
this guy is really flimsy in comparison to other (upper level) icemoon mobs; look at wolves and polar bears for a comparison. also, he doesn't appear to regenerate health like the other upper level icemoon mobs do.